bin/all_sky_search/pycbc_fit_sngls_over_multiparam

```diff
@@ -404,10 +404,12 @@ for param, slog in zip(args.fit_param, args.log_param):
     else:
         raise ValueError("invalid log param argument, use 'true', or 'false'")
 
-nabove_smoothed = []
-alpha_smoothed = []
-ntotal_smoothed = []
-rang = numpy.arange(0, len(nabove))
+n_templates = len(nabove)
+rang = numpy.arange(0, n_templates)
+
+nabove_smoothed = numpy.zeros_like(parvals[0])
```

Comment:
This preallocation saves some time, writing to indexes of the array rather than appending

Reply:
Also on this line, why is `parvals[0]` being used specifically if we have other variables for the number of templates?